app/views/questionnaire.py

```diff
@@ -300,11 +360,19 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
         return redirect(invalid_location.url(metadata))
 
 
+def store_submitted_responses():
+    if g.schema_json.get('view_submitted_responses'):
+        if g.schema_json['view_submitted_responses'].get('enabled'):
```

Comment:
what if enabled is set to `False`?

Reply:
Assuming it returns a boolean then you can just return this line? rather than it being an if statement.